gimme

```diff
@@ -95,21 +96,21 @@ _binary() {
 	urls=(
 		"${GIMME_DOWNLOAD_BASE}/go${version}.${GIMME_OS}-${arch}.tar.gz"
 	)
-	if [ "${GIMME_OS}" = 'darwin' -a "${GIMME_BINARY_OSX}" ] ; then
+	if [ "${GIMME_OS}" = 'darwin' ] && [ -n "${GIMME_BINARY_OSX}" ]; then
 		urls=(
 			"${GIMME_DOWNLOAD_BASE}/go${version}.${GIMME_OS}-${arch}-${GIMME_BINARY_OSX}.tar.gz"
 			"${urls[@]}"
 		)
 	fi
-	if [ "${arch}" = 'arm' ] ; then
+	if [ "${arch}" = 'arm' ]; then
 		# attempt "armv6l" vs just "arm" first (since that's what's officially published)
 		urls=(
-			"${GIMME_DOWNLOAD_BASE}/go${version}.${GIMME_OS}-${arch}v6l.tar.gz" # go1.6beta2 & go1.6rc1
-			"${GIMME_DOWNLOAD_BASE}/go${version}.${GIMME_OS}-${arch}6.tar.gz" # go1.6beta1
-			"${urls[@]}"
+			"${GIMME_DOWNLOAD_BASE}/go${version}.${GIMME_OS}-${arch}v6l.tar.gz"
+			"${GIMME_DOWNLOAD_BASE}/go${version}.${GIMME_OS}-${arch}6.tar.gz"
+			"${urls[@]}" # go1.6beta2 & go1.6rc1 # go1.6beta1
```

Comment:
I'd say it was a bug in my understanding more than anything I'd call a bug in `shfmt` :heart: